src/normalize.css

```diff
@@ -0,0 +1,5 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
```

Comment:
The CSS rule `* { margin: 0; ... }` sets the margin of every element on a web page to 0. Setting the margin of every element to 0 can potentially impact performance by increasing the time it takes for the browser to render the page, particularly if there are a large number of elements on the page.